libs/components/src/modal/modal.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  [ngClass]="width"
+  class="tw-max-h-screen tw-my-4 tw-flex tw-flex-col tw-bg-text-contrast tw-border tw-border-solid tw-border-secondary-300 tw-rounded tw-text-main tw-overflow-hidden"
+>
+  <div
+    class="tw-flex tw-gap-4 tw-p-4 tw-border-0 tw-border-b tw-border-solid tw-border-secondary-300 tw-text-main"
+  >
+    <h2 class="tw-text-lg tw-uppercase tw-flex tw-grow tw-mb-0">
```

Comment:
Is display flex needed here?
```suggestion
    <h2 class="tw-text-lg tw-uppercase tw-grow tw-mb-0">
```

Reply:
resolved in https://github.com/bitwarden/clients/pull/3053/commits/938f2362c707729d900ffabad29b2289efe7fd5a